<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FigureWindow</class>
 <widget class="QMainWindow" name="FigureWindow">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2000</width>
    <height>1600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTreeWidget" name="treeWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="columnCount">
        <number>1</number>
       </property>
       <attribute name="headerMinimumSectionSize">
        <number>15</number>
       </attribute>
       <column>
        <property name="text">
         <string/>
        </property>
       </column>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QGridLayout" name="grid">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2000</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <widget class="QMenu" name="menuColumns">
     <property name="title">
      <string>Columns</string>
     </property>
     <addaction name="actionColumns1"/>
     <addaction name="actionColumns2"/>
     <addaction name="actionColumns3"/>
     <addaction name="actionColumns4"/>
    </widget>
    <widget class="QMenu" name="menuRows">
     <property name="title">
      <string>Rows</string>
     </property>
     <addaction name="actionRows1"/>
     <addaction name="actionRows2"/>
     <addaction name="actionRows3"/>
     <addaction name="actionRows4"/>
    </widget>
    <addaction name="actionGrid"/>
    <addaction name="actionLegend"/>
    <addaction name="actionHold"/>
    <addaction name="menuColumns"/>
    <addaction name="menuRows"/>
   </widget>
   <addaction name="menuView"/>
  </widget>
  <action name="actionGrid">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Grid</string>
   </property>
  </action>
  <action name="actionLegend">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Legend</string>
   </property>
  </action>
  <action name="actionHold">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Hold</string>
   </property>
  </action>
  <action name="actionColumns1">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </action>
  <action name="actionColumns2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </action>
  <action name="actionColumns3">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </action>
  <action name="actionColumns4">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </action>
  <action name="actionRows1">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </action>
  <action name="actionRows2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </action>
  <action name="actionRows3">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </action>
  <action name="actionRows4">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
